README.md

```diff
@@ -103,6 +103,22 @@ If you need some more flexibility, you can completely exclude some files from co
 extend-exclude = ["localized/*.po"]
 ```
 
+### Supported config files
+
+The default option is setting the configuration in `typos.toml`, `_typos.toml` or `.typos.toml`. 
+
+In addition, you can set the same configuration with standard python's config file `pyproject.toml` by adding `[tool.typos.xyz]` where the `xyz` is the same as in default `_typos.toml` so the sample configuration above will be:
+```toml
+[tool.typos.default]
+extend-ignore-identifiers-re = ["AttributeID.*Supress.*"]
+
+[tool.typos.default.extend-identifiers]
+AttributeIDSupressMenu = "AttributeIDSupressMenu"
+
+[tool.typos.default.extend-words]
+teh = "teh"
+```
+
 ### Integrations
```

Comment:
FYI this was covered in #826.  If we want to expand it, we should probably do it in that area within the context of whats there

Reply:
I may suggest to have it readme as it is easy to find and the other location was not discoverable =( so I found it based on issue and merged PR...

Reply:
The README is focused on being a quick start, delegating to this file for the details.

Reply:
Yes but for some cases like us, the need to add a new config in the root could be a deal breaker so in such cases we cannot use it...